actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -17,6 +17,7 @@ class Railtie < Rails::Railtie # :nodoc:
     config.action_dispatch.encrypted_cookie_salt = 'encrypted cookie'
     config.action_dispatch.encrypted_signed_cookie_salt = 'signed encrypted cookie'
     config.action_dispatch.perform_deep_munge = true
+    config.action_dispatch.asset_headers = { }
```

Comment:
added `config.public_file_server` namespace, but will send a separate pull request for adding `config.public_file_server.enabled`.